src/modules/job-list/list.c

```diff
@@ -226,6 +226,11 @@ void list_cb (flux_t *h, flux_msg_handler_t *mh,
         errno = EPROTO;
         goto error;
     }
+    if (since < 0.) {
```

Comment:
commit message suggestion: "ensure since is >= 0" doesn't parse without more context. Suggest "ensure since parameter >= 0"?